examples/young-construct/index.ts

```diff
@@ -0,0 +1,154 @@
+import * as cdk from 'aws-cdk-lib';
```

Comment:
Im not sure why you would need this? I think you should move this Construct to Blueprints Pattern repo.

Reply:
Also purpose of this is unclear

Reply:
This construct is my own for testing. Removed.